deploy_from_github_zip.bash

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+[ $# -eq 0 -o $# -gt 3 ] && {
+    echo "Usage: deploy_from_github_zip.bash <new project name> [GitHub-user/repository-name] [branch]"
+    exit 1
+}
+
+project=$1
+repo=${2-LSSTDESC/start_paper}
+branch=${3-master}
```

Comment:
Yes, the script is general. But, in normal circumstances, the second and third arguments are completely optional, so I don't see that as a bug. We just need to change the default for the 3rd to whatever branch we end up deploying start_paper from. We can customize how the project=folder name comes about however you want.

Also, yes, according to our new plan this script would not be included along with the paper-starting files.